test/types/string.js

```diff
@@ -561,7 +551,8 @@ describe('string', () => {
             Helper.validate(schema, [
                 ['joe@example.com', true],
                 ['joe@www.example.com', true],
-                ['joe@localhost', false, null, '"value" must be a valid email'],
```

Comment:
It seems like a bugfix to me - users who do not desire this behavior can add `localhost` to their `tldBlacklist`.